src/App.js

```diff
@@ -1,23 +1,280 @@
-import logo from './logo.svg';
+// ***********************
+// *** Modules & Hooks ***
+// ***********************
+import { useState } from "react";
+import { citiesList } from "./modules/citiesList";
+
+// ***********************
+// ***** Components ******
+// ***********************
+import ComputerChoice from "./Components/ComputerChoice";
+import PlayerChoice from "./Components/PlayerChoice";
+
+// ***********************
+// ******* Styles ********
+// ***********************
 import './App.css';
 
 function App() {
+
+  // ***************************************
+  // Declaring useStates used in main App.js
+  // ***************************************
+  const [cityChoice, setCityChoice] = useState("Halifax");
+  const [flag, setFlag] = useState("");
+  const [randomCity, setRandomCity] = useState(citiesList[Math.floor(Math.random() * citiesList.length)].city);
```

Comment:
consider doing this logic outside the state, store it in a variable and place that variable into your state